### ISSUE TITLE
failed after 1 try

### ISSUE THREAD

**Issue by Beroyale** [chinoogawa/instaBrute#8]
its gives you the first password , failed

**Comment by Coto16** [comment]
There is an upcoming update coming soon. As this account-cracking method has been busted by Instagram, the Dev, @chinoogawa is working on an update, where you must import a Proxy (IP:Port) list in order for the Module to run successfully. So, all we all have to do, is just wait a little while.

**Comment by Nas-au** [comment]
@Coto16 @chinoogawa  Hey guys any update on the proxy implementation to the code?

**Comment by Coto16** [comment]
@Nas-au Nope, nothing I've heard of. I'll be working on a Proxy solution now that I have seen how busy @chinoogawa is... But remember, that instaBrute was made for that reason, that they had almost no protection against bruteforcing. Looks like they have, so I need to make a Proxy implementation to the code. For more updates, add me on Skype: iLouisy

**Comment by aleperno** [comment]
I dont promise it will be done son.

- [ ] Validate if method requires mods.
- [ ] Implement some way of proxy

**Comment by Coto16** [comment]
> I dont promise it will be done soon.

Agree, however, I've used this script to make numerous other things. So still, the work of the Dev didn't go a waste, not for me at least. :smile:

**Comment by aleperno** [comment]
@Coto16 Could you be more specific on which error the current method has? I've made very (VERY) simple tests and seems it is still a valid method.

About proxy support, I am currently working on it, however it will not be a mod of the current version but rather a complete different approach. I wont push changes to the master branch.

**Comment by Coto16** [comment]
@aleperno From what I remember, yes, it does work for Instagram, until it it blocks you for too many tries, meaning you need to switch between proxies.

Please contact me on Skype for more Info: Null0Sec or coto993

**Comment by aleperno** [comment]
Simple implementation at 4f3ad08b2cb955546d956a308593ebc93e2ef0c4
Requires python yaml module (pyyaml).
I included a simple example (I used tor), ports to other sockets or http proxies should be trivial. 
Also note a bug to be solved, since when using a proxy the process may take longer than expected, instaBrute tries with another password before receiving the last request answer. Hence it may show no correct password, or show the wrong one.
